### ISSUE TITLE
Penn State Theme - Buttons have Insufficient Color Contrast

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#620]
BUTTONS: Element has insufficient color contrast of 1.90 (foreground color: #ffffff, background color: #66cccc, font size: 10.5pt, font weight: normal)

`<a href="http://www.psupress.org/books/titles/978-0-271-07576-1.html" target="_blank" title="Buy Nothing but Love in God’s Water: Volume 2: Black Sacred Music from Sit-Ins to Resurrection City" role="button" class="btn btn-default btn-small">Buy Book</a>`

`#main > div:nth-of-type(2) > .col-md-3.sidebar > .monograph-info > .monograph-metadata > .btn-toolbar > .btn-group > .btn.btn-default.btn-small`

- [ ] 1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: (Level AA)
Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.

**Comment by jmcglone** [comment]
Example:

![screen shot 2017-04-25 at 3 20 11 pm](https://cloud.githubusercontent.com/assets/1686111/25403566/b8f2c4e4-29ca-11e7-836a-7e811042d1df.png)

@mbakeryo it might be worth bringing this to Patrick's attention – the button colors are based on the mockup/style guide they gave us and has been implemented elsewhere (Ubiquity Press). One way to fix this would be to use a darker color for the text such as the navy blue color used for links.

**Comment by jmcglone** [comment]
MBY shared this information with Patrick. It is blocked until Patrick responds with how he'd like to proceed.

**Comment by mbakeryo** [comment]
4/28/17: Sent email to Patrick, cc'd @jmcglone

**Comment by mbakeryo** [comment]
5/9/17: Jennifer's feedback: Thanks for checking in on this. Things are looking great! With regard to the color contrast issue, could we use dark gray for more contrast? I see though, that the rollover color is dark blue, so I’m not sure if the text color will change also with the rollover, or if we could simply change the rollover color to a bit darker teal so then we could keep the dark gray text. I’m hoping that makes sense. 

One other design issue that I’m sorry I didn’t write about sooner. I think the serif text face used is a little hard to read (the copy on the home page under the head “Robert Darden” and also used for the book entries. Would it be possible to use the same text font that is being used for Northwestern and Minnesota while keeping the sans serif just as is?

**Comment by jmcglone** [comment]
Clean up the external resources icon on the gallery page
